README.md

```diff
@@ -232,7 +233,7 @@ export const hipo = {
   peso: 80,
   hambre: 0,
   puedeParticiparDeUnaPosta(posta) {
-    return this.barbarosidad >= 10 && this.__proto__.puedeParticiparDeUnaPosta(posta)
+    return this.barbarosidad >= 10 && super.puedeParticiparDeUnaPosta(posta)
```

Comment:
Cuando un objeto delega en un prototipo, tiene disponible la palabra clave `super`.
De otra manera habría que usar `Object.getPrototypeOf(this)` más que `__proto__` como propiedad, que está deprecado.

Reply:
¡mirá qué piola! esto sí que no lo conocía, al menos en js5 no conocía el uso de `super` (sí sabía que el uso de `__proto__` no era uniforme en todas las implementaciones)